util/mkdef.pl

```diff
@@ -212,6 +214,8 @@
 foreach (@ARGV, split(/ /, $config{options}))
 	{
 	$debug=1 if $_ eq "debug";
+	$trace=1 if $_ eq "trace";
+	$warn_simp=1 if $_ eq "warn_simp";
```

Comment:
Why not just "warn"?

Reply:
Because there are also other types warnings that seem to be taken more serious, for instance in case there are duplicate or missing symbol entries in the table.

Reply:
Then, maybe "verbose"?

Reply:
Sounds good to me; done. 
I made use of the new `verbose` option also for the following: 

		print STDERR "Warning: $num_noinfo symbols were without info." if $verbose || !$do_rewrite;
		if ($do_rewrite) {
			printf STDERR "  The rewrite will fix this.\n" if $verbose;
		} else {
			printf STDERR "  You should do a rewrite to fix this.\n";
		}